src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -123,22 +132,10 @@ public void Constructor_BoundSolution_SyncsAndStartsMonitoring()
         {
             // Arrange
             SetupSolutionBinding(isBound: true, isConnected: true, projectKey: "keyXXX", issues: new List<SonarQubeIssue>());
-            EventWaitHandle waitHandle = new EventWaitHandle(false, EventResetMode.ManualReset);
```

Comment:
I've changed the _SetupSolutionBinding_ method to automatically set up the wait handle for all bound solutions